source/Calamari.Tests/Fixtures/PowerShell/PowerShellFixture.cs

```diff
@@ -18,15 +18,34 @@ namespace Calamari.Tests.Fixtures.PowerShell
 {
     [TestFixture]
     [Category(TestCategory.CompatibleOS.Windows)]
-    public class PowerShellCoreOnWindows : PowerShellFixture
+    public class WindowsPowerShellCoreFixture : PowerShellFixture
     {
         protected override PowerShellEdition PowerShellEdition => PowerShellEdition.PowerShellCore;
+
+        bool IsPowerShellInstalled()
+        {
+            var (output, _) = RunScript("PSCoreVersion.ps1");
```

Comment:
`RunScript` runs a powershell script through `Calamari`.

- If PS Core is installed and everything is working, then the script will return `PowerShell x` in the output and the test will continue ✔️ 
- If PS Core is not installed and everything is working normally, then the script will not return `PowerShell x` in the output an the test will return an inconclusive result ✔️ 
- If PS Core is installed and something isn't working correctly (eg. a bug in calamari which causes calamari to crash) then the script will not include `Powershell x` in its output, and the test will return an inconclusive result 🔴.

This last case is bad because it could mask real issues.

It would be better to not use calamari at all when it comes to detecting if powershell is installed a pre-requisite of the test.

Are you familiar with the Arrange, Act, Assert pattern of writing unit tests? We only really want to invoke production code (in this case calamari) during the `Act` phase of the test. This part of the test (checking if powershell is installed) is part of the `Arrange` phase and should not use Calamari in any way, so as to be a more stable test.